xml/System.Windows.Media/PenDashCap.xml

```diff
@@ -13,7 +13,7 @@
     <BaseTypeName>System.Enum</BaseTypeName>
   </Base>
   <Docs>
-    <summary>Describes the shape at the ends of each dash segment.</summary>
+    <summary>Describes the shape at the end of each dash segment.</summary>
```

Comment:
Minor nitpick here: both ends of each dash are affected by the cap setting.

Reply:
Which cap setting are you referring to? Should it be included as a remark?

Reply:
PenDashCap (this enum) is the shape (the line cap) at the ends of each dash segment.  If that doesn't seem clear, 'line cap' can be added to the main description: 

 <summary>Describes the shape (the line cap) at the ends of each dash segment.</summary>